ui.apps/src/main/content/jcr_root/apps/aemtraining/templates/page-content/.content.xml

```diff
@@ -3,28 +3,9 @@
     jcr:description="Template with focus on content."
     jcr:primaryType="cq:Template"
     jcr:title="AEMDemoSite Content Page"
-    allowedPaths="[/content/aemtraining/.+]"
+          allowedPaths="[/content/aemtraining/us/en/.+,/content/aemtraining/us/en/home/.+,/content/aemtraining/.+]"
```

Comment:
The `/content/aemtraining/.+` regpression fully govers the `/content/aemtraining/us/en/.+` expression and the `/content/aemtraining/us/en/home/.+` expression.
These are just reuglar expressions.
Therefor, only the  `/content/aemtraining/.+`  part is needed